templates/mod/security_crs.conf.erb

```diff
@@ -1,4 +1,4 @@
-<% if scope['facts']['os']['family'] == 'RedHat' and scope['facts']['os']['release']['major'].to_i <= 7 -%>
+<% if scope['facts']['os']['family'] == 'RedHat' and scope['facts']['os']['release']['major'].to_i == 7 -%>
```

Comment:
I'd keep this the same. It doesn't make any difference and fewer changes in git is easier to review.
```suggestion
<% if scope['facts']['os']['family'] == 'RedHat' and scope['facts']['os']['release']['major'].to_i <= 7 -%>
```